package.json

```diff
@@ -59,6 +59,7 @@
     ]
   },
   "dependencies": {
+    "@jsbits/get-package-version": "^1.0.3",
```

Comment:
@snormore Got to this one a bit late, but I think you can replace this external dependency with
`import { version } from '../package.json'` in Typescript, so long as the tsconfig has the `resolveJsonModule` turned on.